doc/edge_orchestration_api_secure.yaml

```diff
@@ -118,13 +118,13 @@ definitions:
   verifier:
     required:
       - SecCompName
```

Comment:
Is `SecCompName` right?
The example of curl use `SecureMgr` instead of `SecCompName`.
Please check out other APIs too.

Reply:
@t25kim  Done! Thank you!